pkgs/top-level/all-packages.nix

```diff
@@ -159,6 +159,10 @@ in
 
   demoit = callPackage ../servers/demoit { };
 
+  deviceTree = callPackage ../os-specific/linux/device-tree {};
```

Comment:
No camelCase in package attributes please.

Reply:
`deviceTree` is not a package, it currently only contains the `applyOverlays` attribute, which is for applying overlays to derivations containing device-tree files. Should this still be renamed?

`deviceTree_rpi` is a package, I will rename this to `device-tree_rpi` (to parallel `linux_rpi`) unless somebody suggests a better name.